src/Lib/Canvas.vala

```diff
@@ -124,9 +132,98 @@ public class Akira.Lib.Canvas : Goo.Canvas {
         events |= Gdk.EventMask.BUTTON_PRESS_MASK;
         events |= Gdk.EventMask.BUTTON_RELEASE_MASK;
         events |= Gdk.EventMask.POINTER_MOTION_MASK;
+
+        update_bounds ();
```

Comment:
Remove this as well.
Also, there's no need to call this method on `construct` if you're calling it in the MainCanvas file as you're basically tapping the same method twice.